modules/osd-create-cluster-red-hat-account.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * osd_quickstart/osd-quickstart.adoc
+// * osd_cluster_create/creating-an-aws-cluster.adoc
+// * osd_cluster_create/creating-a-gcp-cluster.adoc
+
+ifeval::["{context}" == "osd-getting-started"]
+:osd-quickstart:
+endif::[]
+ifeval::["{context}" == "osd-creating-a-cluster-on-aws"]
+:osd-on-aws:
+endif::[]
+ifeval::["{context}" == "osd-creating-a-cluster-on-gcp"]
+:osd-on-gcp:
+endif::[]
+
+:_content-type: PROCEDURE
+ifdef::osd-quickstart[]
+[id="create-cluster-red-hat-account_{context}"]
+= Creating a cluster with a Red Hat cloud account
+endif::osd-quickstart[]
+ifdef::osd-on-aws[]
+[id="osd-create-aws-cluster-red-hat-account_{context}"]
+= Creating a cluster on AWS with a Red Hat cloud account
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+[id="osd-create-aws-cluster-ccs_{context}"]
+= Creating a cluster on GCP with a Red Hat cloud account
+endif::osd-on-gcp[]
+
+Through {OCM}, you can create an {product-title} cluster 
+ifdef::osd-on-aws[]
+on {AWS} 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+on {GCP} 
+endif::osd-on-gcp[]
+using a standard cloud provider account owned by Red Hat.
+
+.Procedure
+
+. Log in to {console-redhat-com} and click *Create Cluster*.
+
+. In the *Cloud* tab, click *Create Cluster* in the *Red Hat OpenShift Dedicated* row.
+
+. In the *Billing model* dialog, configure the subscription type and infrastructure type:
+.. Select a subscription type.
+.. Select the *Red Hat cloud account* infrastructure type to deploy {product-title} in a cloud provider account that is owned by Red Hat.
+.. Click *Next*.
+
+ifdef::osd-quickstart[]
+. Select *Run on Amazon Web Services* or *Run on Google Cloud Platform* to choose a cloud infrastructure provider 
+endif::osd-quickstart[]
+ifdef::osd-on-aws[]
+. Select *Run on Amazon Web Services* 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+. Select *Run on Google Cloud Platform* 
+endif::osd-on-gcp[]
+and click *Next*.
+
+. On the *Cluster details* page, provide a name for your cluster and specify the cluster details:
+.. Add a *Cluster name*.
+.. Select a cluster version from the *Version* drop-down menu.
+.. Select a cloud provider region from the *Region* drop-down menu.
+.. Select a *Single zone* or *Multi-zone* configuration.
+.. Select a *Persistent storage* capacity and state the number of *Load balancers* that you require for your cluster.
+.. Leave *Enable user workload monitoring* selected to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. The option is enabled by default.
```

Comment:
same suggestion: `s/The option/This option`

Reply:
Resolved.